examples/default.yaml

```diff
@@ -61,11 +61,28 @@ mounts:
     # system will look and feel like regular files directories in the Guest OS.
     # 🟢 Builtin default: false
     followSymlinks: null
+  virtfs:
+    # Supported security models are "passthrough", "mapped-xattr", "mapped-file" and "none".
+    # 🟢 Builtin default: "mapped-xattr"
+    securityModel: null
+    # Select 9P protocol version. Valid options are: "9p2000" (legacy), "9p2000.u", "9p2000.L".
+    # 🟢 Builtin default: "9p2000.L"
+    protocolVersion: null
+    # The number of bytes to use for 9p packet payload, 4 KiB is minimum and 8 KiB is default.
+    # 🟢 Builtin default: "128 KiB"
```

Comment:
This is confusing: is the default 8 or 128? Do we have to mention the `8 KiB` default at all?

Is there any advantage to making the size a multiple of some block size, like 512, or 4 KiB?

Reply:
The 8 KiB default is in older kernels and some clients, but you get a warning about bad performance so we increase a bit

Reply:
https://lists.gnu.org/archive/html/qemu-devel/2012-01/msg01680.html

Reply:
> The 8 KiB default is in older kernels and some clients, 

Yeah, but what value does this information provide to the user? I understand that the minimum size is 4 KiB and the Lima default is 128 KiB. What do I do with the information that some other (unspecified) default is 8 KiB? If it is not helpful to the end-user, it should be removed from the docs to avoid confusion